playbooks/roles/ocp-install/tasks/main.yaml

```diff
@@ -54,3 +57,22 @@
     src: "{{ workdir }}/auth/kubeconfig"
     dest: "~/.kube/config"
     force: yes
+
+- name: Create admin authentication using htpasswd
+  tags: admin-user
+  include: templates/create_oauth.yaml
```

Comment:
Can have a template task here instead of nesting via a template file.

Reply:
Or create files/ directory in the role and move this yaml file there if it is just copy/paste.